ts/utils/payment.ts

```diff
@@ -307,3 +317,21 @@ export const getPaymentOutcomeCodeDescription = (
   }
   return none;
 };
+
+export const getBancomatOrCreditCardPickMethodDescription = (
+  bancomatOrCreditCard:
+    | CreditCardPaymentMethod
+    | PrivativePaymentMethod
+    | BancomatPaymentMethod
+) => {
+  const translatedExpiryDate = getTranslatedShortNumericMonthYear(
```

Comment:
```suggestion
  const translatedExpireDate = getTranslatedShortNumericMonthYear(
```

Reply:
Done in [77f453b](https://github.com/pagopa/io-app/pull/2926/commits/77f453b8c8d136c3d17727e0de209212384f3a2e)